src/test/java/io/redisearch/client/ClientTest.java

```diff
@@ -1137,7 +1137,12 @@ public void testConfig() throws Exception {
         cl.setConfig(ConfigOption.ON_TIMEOUT, "fail");
         assertEquals("fail", cl.getConfig(ConfigOption.ON_TIMEOUT));
 
-        assertFalse(cl.setConfig(ConfigOption.ON_TIMEOUT, "null"));
+        try {
+            cl.setConfig(ConfigOption.ON_TIMEOUT, "null");
+        }catch(JedisDataException e) {
+            assertTrue(true);
+            return; // Starting from RediSearch 2.0 this command is not supported anymore
```

Comment:
What's the point of `assertTrue(true)`? Maybe should `AssertTrue(false)` after `cl.setConfig(...)` (if exception wasn't raised)?